README.md

```diff
@@ -112,6 +114,24 @@ have long served the same purpose, and are easily accessible within a
 As such, `config-loader` does not call `Function` configs with the `env`
 parameter. Rather, it makes calls with only the `argv` parameter.
 
+### Using extends with symlinked modules
+
+When using `extends` to extend a configuration which exists in a different package, care must be taken to ensure you don't hit module resolution issues if you are developing with these packages symlinked (i.e. with `npm link` or `yarn link`).
+
+By default, node does not search for modules through symlinks, and so you may experience errors such as this:
+
+`module not found: Error: Can't resolve 'webpack-hot-client/client'`
+
+This can be fixed by using node's `--preserve-symlinks` flag which will allow you to develop cross-module without experiencing inconsistencies when comparing against a normal unlinked install:
+
+For webpack-command:
+
+`node --preserve-symlinks ./node_modules/.bin/wp`
```

Comment:
let's wrap this in a three-tick fence with the `console` type, same for the one below